opm/simulators/linalg/ISTLSolverBda.cpp

```diff
@@ -78,12 +78,15 @@ void BdaSolverInfo<Matrix,Vector>::
 prepare(const Grid& grid,
         const Dune::CartesianIndexMapper<Grid>& cartMapper,
         const std::vector<Well>& wellsForConn,
+        //const std::unordered_map<std::string, std::set<std::array<int,3>>>& possibleFutureConnections,
         const std::vector<int>& cellPartition,
         const std::size_t nonzeroes,
         const bool useWellConn)
 {
+    std::unordered_map<std::string, std::set<std::array<int,3>>> possibleFutureConnections;
```

Comment:
I think we should fix this. Currently, we just silently assume that no extra connections will be defined via ACTIONX.

Hopefully, we can get the futureConnections at the [calling site ISTLSolverBda.hpp#L207](https://github.com/OPM/opm-simulators/blob/9f76155312d1c5f96c0be15bcd9205854c0251a0/opm/simulators/linalg/ISTLSolverBda.hpp#L207) and pass it as an argument.